orchestra/communication/tests/test_staffing.py

```diff
@@ -36,6 +40,16 @@ def test_handle_staffing_response_is_available(self):
         self.assertTrue(response.is_winner)
         self.assertEqual(StaffingResponse.objects.all().count(), old_count + 1)
 
+        task_assignment = (TaskAssignment.objects
```

Comment:
i don't understand what this test does, shouldn't you check the state of the object before and after the request is made?